### ISSUE TITLE
[desktop] On the Chinese input of TextFiled

### ISSUE THREAD

**Issue by nukiyoam** [JetBrains/compose-jb#1615]
When using TextField, after entering Chinese, sometimes the following errors will appear when deleting one or more texts.

>ComposeVesion: 1.0.1-rc2
>JDK: Oracle jdk11
>OS: mac bigsur 12

```text
java.lang.reflect.InvocationTargetException
	at java.desktop/sun.lwawt.macosx.LWCToolkit.checkException(LWCToolkit.java:741)
	at java.desktop/sun.lwawt.macosx.LWCToolkit.invokeAndWait(LWCToolkit.java:710)
	at java.desktop/sun.lwawt.macosx.CInputMethod.attributedSubstringFromRange(CInputMethod.java:590)
Caused by: java.lang.StringIndexOutOfBoundsException: begin 50, end 51, length 50
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3319)
	at java.base/java.lang.String.substring(String.java:1874)
	at java.base/java.lang.String.subSequence(String.java:1913)
Caused by: java.lang.StringIndexOutOfBoundsException: begin 50, end 51, length 50
```

> There are other related problems.
> [https://github.com/JetBrains/compose-jb/issues/672#issuecomment-985376744](https://github.com/JetBrains/compose-jb/issues/672#issuecomment-985376744)

**Comment by igordmn** [comment]
Should be fixed in 1.1.0

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-5116">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
